lib/std/jsbigints.nim

```diff
@@ -8,12 +8,14 @@ type JsBigInt* = distinct JsBigIntImpl        ## Arbitrary precision integer for
 
 func big*(integer: SomeInteger): JsBigInt {.importjs: "BigInt(#)".} =
   ## Constructor for `JsBigInt`.
+  when nimvm: doAssert false, "JsBigInt can not be used at compile-time nor static context" else: discard
```

Comment:
Can the `error` pragma be used here instead?

Reply:
Nope, because it errors out both ways, even with code that should compile. Good observation anyway.

Reply:
same as https://github.com/nim-lang/Nim/pull/17161#discussion_r583124021